tests/send_CAPI_command.py

```diff
@@ -126,29 +193,20 @@ def dev_send_1905(self, dest: str, message_type: int, *tlvs: Tuple[int, int, str
         message_type : int
             The message type of the 1905.1 message to be sent, as an integer.
 
-        tlvs : Tuple[int, int, str]
-            A list of TLV descriptions. Each TLV is a 3-tuple (type, length, value). The type and
-            length are given as integers. The value is given as a string, formatted according to the
-            UCC rules, i.e. with curly braces and hexadecimal numbers.
+        tlvs : tlv
+            Additional arguments are the TLVs in the 1905.1 message, as `tlv` objects.
 
         Returns
         -------
         The MID of the message, as an integer.
         """
-        def format_tlv(tlv_num, tlv_type, tlv_length, tlv_value):
-            if tlv_num:
-                tlv_num = str(tlv_num)
-            else:
-                tlv_num = ''
-            return "tlv_type{tlv_num},0x{tlv_type:02x}," \
-                   "tlv_length{tlv_num},0x{tlv_length:04x}," \
-                   "tlv_value{tlv_num},{tlv_value}".format(**locals())
-
-        cmd = "DEV_SEND_1905,DestALid,{dest:s},MessageTypeValue,0x{message_type:04x}".format(**locals())
+        cmd = "DEV_SEND_1905,DestALid,{dest:s},MessageTypeValue,0x{message_type:04x}"\
+            .format(locals())
```

Comment:
Why did you remove the unpacking? Now the format method receives a dict, and this code should fail...

Reply:
That was an accident, fixed.